manifest_test.go

```diff
@@ -608,6 +608,69 @@ func TestValidateProjectRoots(t *testing.T) {
 	}
 }
 
+func TestFromRawPruneOptions(t *testing.T) {
+	cases := []struct {
+		name            string
+		rawPruneOptions rawPruneOptions
+		wantOptions     gps.RootPruneOptions
+	}{
+		{
+			name: "global all options project no options",
+			rawPruneOptions: rawPruneOptions{
+				UnusedPackages: true,
+				NonGoFiles:     true,
+				GoTests:        true,
+				Projects: []rawPruneProjectOptions{
+					{
+						Name:           "github.com/golang/dep/gps",
```

Comment:
Not actually a legal name, here - it's expected to be a project root (`github.com/golang/dep`). Given the inherent ambiguity of this, even if the test does what we want in the short term, it's better to never give the wrong impression.